documentation/content/Contributing/automated-release.md

```diff
@@ -1,42 +1,93 @@
 ---
-Title: Fully automated version management and publishing
-Date: 2019-07-20 23:17
+Title: Automated Version Management and Publishing
+Subtitle:
 Slug: fully-automated-version-management-and-publishing
 Category: Contributing
-Authors: Talha Mansoor
+Tags:
+Date: 2019-07-20 23:17
+Summary: Elegant utilizes a 'release early, release often' philosophy that embraces a fully automated release process.
+Keywords:
+Authors: Talha Mansoor, Jack De Winter
 ---
 
-Elegant follows "release early, release often" software development philosophy. In this regard, our release process is fully automated.
+[TOC]
+
+The Elegant project follows a "release early, release often" software development philosophy.
+By employing this philosophy, we are able to have each team member working on a different
+aspect of the project with little friction from each other's changes. At any point, any team
+member can pull changes that have been made in another branch to their fork of the repository
+and exercise their changes with complete confidence.
+
+Another benefit of this philosophy if that in order to attain this goal, our release process
```

Comment:
> Another benefit of this philosophy if that in order to attain this goal, our release process
must be fully automated.

Release process should be fully automated to materialize this goal.

Reply:
done.